File: Working/Parser/test/sample.java
/**
 * This is a file header comment
 */

import java.util.*;

public class sample {
    /** This is a variable */
    int member;

    public static int func_no_doc() {
        // Here is a normal comment
        int x;
        /* This is another normal comment */
        int y;
        return 0;
    }

    /**
     * This is the main function
     */
    public static void main(String[] args) {
        return;
    }
}

interface Inter {
    /**Documentation */
    public void f();
}

File: Working/Parser/test/sample.py
"""
This is a file header comment
"""

import numpy as np

def function(a, b):
    """
    This is a function header comment
    """
    return a + b

def func_no_doc(a, b):
    # This is a normal comment
    return a - b

if __name__ == "__main__":
    # This is another normal comment
    function(1, 2)
    fun_no_doc(2, 1)